pyproject.toml

```diff
@@ -75,16 +73,52 @@ Repository = "https://github.com/gchq/coreax"
 Issues = "https://github.com/gchq/coreax/issues"
 Changelog = "https://github.com/gchq/coreax/blob/main/CHANGELOG.md"
 
-[tool.isort]
-known_first_party = ["coreax", "examples"]
-profile = "black"
-
 [tool.setuptools]
 packages = ["coreax"]
 
 [tool.setuptools.dynamic]
 version = {attr = "coreax.__version__"}
 
+[tool.ruff]
+src = [".", "examples"]
+
+[tool.ruff.lint]
+preview = true
+select = [
+    "B",   # flake8-bugbear
+    "C90", # McCabe
+    "D",   # pydocstyle
+    "E",   # pycodestyle [error]
+    "F",   # pyflakes
+    "I",   # isort
+    "N",   # pep8-naming
+    "PL",  # pylint
+    "W"    # pycodestyle [warning]
+]
+ignore = [
+    "D203",
```

Comment:
I know we can look them up, but can you hint towards them in comments please?

Reply:
Do you want hints in the places where noqa is used as well?